### PR TITLE
19091: Phone Application- User Disagrees to Experian Identity Proofing- User directed to Document Upload page

### PR DESCRIPTION
**Pull request by maheshkurma**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [19091](https://devops.dchbx.org/redmine/issues/19091) |
| App-Dev Road Map # | n/a |
| Start Date | n/a |
| Due Date | n/a  |
| App-Dev Priority | UP-1 |
| Development Story Points | 2 |
| Peer Review Story Points | n/a |
| Ticket Author | @karashearer5 |

### CODE REVIEW

File: spec/views/insured/consumer_roles/_application_type_field_spec.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+include ActionView::Context
+RSpec.describe "insured/consumer_roles/_form.html.erb" do
+  context "Hbx Admin shows application type field in Persnoa Information Page" do
+    binding.pry
```
Comment (psk86) [inline spec/views/insured/consumer_roles/_application_type_field_spec.rb]: remove this

File: spec/views/insured/consumer_roles/_application_type_field_spec.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      mock_form = ActionView::Helpers::FormBuilder.new(:person, @person, helper, {})
+      stub_template "shared/_consumer_fields.html.erb" => ''
+      allow(view).to receive(:policy_helper).and_return(double("FamilyPolicy", updateable?: true))
+      assign(:consumer_role, @person.consumer_role)
+      assign(:person, @person)
+      render partial: "insured/consumer_roles/form", locals: {f: mock_form}
+    end
+
+    it "should display the label text Application Type" do
+      binding.pry
```
Comment (psk86) [inline spec/views/insured/consumer_roles/_application_type_field_spec.rb]: remove this too

File: features/step_definitions/phone_application_steps.rb [hunk 3]
```diff
@@ -0,0 +1,92 @@
+And(/^admin has navigated into the NEW CONSUMER APPLICATION$/) do
```
Comment (psk86) [inline features/step_definitions/phone_application_steps.rb]: good cucumber scenarios

File: app/assets/stylesheets/custom.scss.erb [hunk 4]
```diff
@@ -2112,6 +2129,7 @@ input[type="checkbox"] + h4.agree {
     padding-left: 0;
 }
 
+
```
Comment (psk86) [inline app/assets/stylesheets/custom.scss.erb]: remove additional line

File: app/models/person.rb [hunk 5]
```diff
@@ -53,6 +53,8 @@ class Person
   field :updated_by, type: String
   field :no_ssn, type: String #ConsumerRole TODO TODOJF
 
+  field :application_type, type: String
```
Comment (psk86) [inline app/models/person.rb]: remove this. new field not needed, make use of application_type in families

File: app/views/shared/_individual_progress.html.haml [hunk 6]
```diff
@@ -42,13 +42,16 @@
...
+  - if next_link.present?  && current_user.has_hbx_staff_role? && @person.application_type == "Phone"
+    = link_to "Continue", next_link, :method => :get, class: 'button btn btn-lg btn-primary  btn-block interaction-click-control-continue', id: 'btn-continue', :disabled => !application_verified
+  - elsif next_link.present?
     = link_to "Continue", next_link, :method => :get, class: 'button btn btn-lg btn-primary  btn-block interaction-click-control-continue', id: 'btn-continue', :disabled => (current_user.has_hbx_staff_role? ? false : !(identity_verified  && application_verified))
   - else
     = link_to "Continue", new_insured_interactive_identity_verification_path, class: 'button btn btn-lg btn-primary  btn-block interaction-click-control-continue', id: 'btn-continue'
-
 - elsif step == 2
-  - if next_link.present?
+  - if next_link.present?  && current_user.has_hbx_staff_role? && @person.application_type == "Phone"
```
Comment (psk86) [inline app/views/shared/_individual_progress.html.haml]: application_type for families : @person.primary_family.application_type

**Comment by harshared** [comment]
already merged and closing

**Pull request closed without merging**